package.json

```diff
@@ -86,6 +86,7 @@
     "**/istanbul-lib-coverage": "^3.2.0",
     "**/json-schema": "^0.4.0",
     "**/minimist": "^1.2.5",
+    "**/node-jose": "^1.1.4",
```

Comment:
As mentioned in ad696ffc8c30514f3a53e3ab023ce33e41c250e5, I had to use a yarn dependency resolution to upgrade this transitive dependency.

We depend on `node-jose` via `@elastic/request-crypto`, so I submitted another PR there to upgrade that dependency: elastic/request-crypto#11

Once that is merged and a new version is deployed, I can remove this dependency resolution. I wasn't sure how long that would take but it looks like that might be done tomorrow, so I might be able to upgrade that instead and remove this dependency resolution before merging this PR.

Reply:
Both options sounds good to me :+1:

Reply:
Updated to @elastic/request-crypto 2.0.0 and node-jose 2.0.0, and removed the dependency resolutions, in fa4eecca5b657c5cf6709e760a295561290ad4dc.